### PR TITLE
Add gif of Semgrep extension

### PR DESCRIPTION
**Pull request by adamkvitek**
PR checklist:

- [ ] Purpose of the code is [evident to future readers](https://semgrep.dev/docs/contributing/contributing-code/#explaining-code)
- [ ] Tests included or PR comment includes a reproducible test plan
- [ ] Ran tests locally (VSCode tests cannot run in CI)
- [ ] Documentation is up-to-date
- [ ] A changelog entry was for any user-facing change
- [ ] Change has no security implications (otherwise, ping security team)

If you're unsure about any of this, please see:

- [Contribution guidelines](https://semgrep.dev/docs/contributing/contributing-code)!
- [One of the more specific guides located here](https://semgrep.dev/docs/contributing/contributing/)

### CODE REVIEW

**Review by ajbt200128 — APPROVED** [review]
LGTM

File: assets/semgrep-extension.gif
Comment (underyx) [inline assets/semgrep-extension.gif:1]: how come this is so slow? results should appear within ~100ms, no? cc @ajbt200128

Comment (ajbt200128) [reply assets/semgrep-extension.gif:1]: We currently only scan on save, it's subtle but the user did not save for a few seconds after uncommenting the line. I believe Raja is making a new gif hence why this PR is still open.

**Comment by ajbt200128** [comment]
Closed in favor of #54

**Pull request closed without merging**